test/wasmgc/Pointer_NULL0_gc.v3

```diff
@@ -0,0 +1,6 @@
+//@execute = true
```

Comment:
We can probably land this for now, but I think it'd be better to add these to the pointer tests and then mark the ones that don't work with Wasm-GC with a comment (or put them in a file).